sdk/storage/azure-storage-queue/CHANGELOG.md

```diff
@@ -1,20 +1,9 @@
 # Release History
 
-## 12.13.0-beta.2 (Unreleased)
-
-### Features Added
-- Added support for 2021-06-08 service version.
-
-### Breaking Changes
-
-### Bugs Fixed
-
-### Other Changes
+## 12.13.0-beta.2 (2022-05-24)
```

Comment:
This is a GA release, please change this to 12.13.0.

Reply:
pom file - https://github.com/Azure/azure-sdk-for-java/blob/main/sdk/storage/azure-storage-queue/pom.xml#L16

Reply:
Dependency updates?